docs/tutorials/neutron-diffraction.ipynb

```diff
@@ -34,6 +34,8 @@
     "[PG3_4866_event.nxs](http://198.74.56.37/ftp/external-data/MD5/3d543bc6a646e622b3f4542bc3435e7e).\n",
     "Both are available as part of Mantid's test data.\n",
     "\n",
+    "Rename the files upon download.\n",
```

Comment:
Maybe better to refer to `data/fetch_neutron_data.py` and change back to PG3_4844